prov/sockets/src/sock_ep_msg.c

```diff
@@ -1093,11 +1091,6 @@ static void *sock_pep_listener_thread(void *data)
 						conn_req->hdr.msg_id);
 
 		switch (conn_req->hdr.type) {
-		case SOCK_CONN_ACK:
-			SOCK_LOG_DBG("Received SOCK_CONN_ACK\n");
-			sock_ep_cm_handle_ack(&pep->cm, &conn_req->hdr);
-			break;
-
```

Comment:
This change is unrelated.  Why was this code removed?  Is this just an invalid message to process at this point?